tasks/prime.py

```diff
@@ -272,7 +274,80 @@ def on_start(self):
             name="/internal/service_members/{serviceMemberId}/backup_contacts",
             data=json.dumps(payload),
             headers={"content-type": "application/json"},
-            **self.user.cert_kwargs,
+        )
+
+        # Setup customer order
+        overrides = {
+            "service_member_id": service_member_id,
+            "issue_date": datetime.now().strftime("%Y-%m-%d"),
+            "report_by_date": datetime.now().strftime("%Y-%m-%d"),
+            "orders_type": "PERMANENT_CHANGE_OF_STATION",
+            "orders_type_detail": "HHG_PERMITTED",
+            "has_dependents": False,
+            "spouse_has_pro_gear": False,
+            "new_duty_station_id": stations[1]["id"],
```

Comment:
Do we know what this duty station will be as the second result?

Reply:
The first duty station is `MCAGCC Twentynine Palms` and the second is `29 Palms CA`. When running the service locally you can go to `http://milmovelocal:3000/internal/duty_stations?search=palms`